CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
     Bug #4746: Non-solid player can't run or sneak
     Bug #4747: Bones are not read from X.NIF file for NPC animation
     Bug #4750: Sneaking doesn't work in first person view if the player is in attack ready state
+    Bug #4763: Setting scale of the player confuses NPC pathfinding
```

Comment:
Usually we do not add regression fixes to the changelog.

Reply:
Ok. I will remove it. Does `Regression` tag in gitlab means this issue shouldn't be add to changelog? Or is there any other formal condition?